service-commands/scripts/generateWhiteNoise.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Generates a random generateWhiteNoise mp3 file of given input size.
+ *
+ * Script usage: node generateWhiteNoise.js <size> <outFile>
+ */
+
+const { spawn } = require('child_process')
+const ffmpeg = require('ffmpeg-static')
+
+async function run() {
+  try {
+    const { size, outFile } = parseArgs()
+    generateWhiteNoise(size, outFile)
+  } catch (err) {
+    console.error(err)
+  }
+}
+
+/**
+ * Process two command line args for filesize and filename.
+ *
+ * Defaults to 1 MB file size, and unix timestamp for filename if args are not provided.
+ */
+function parseArgs() {
+  const args = process.argv.slice(2)
+  const size = args[0] || 1000
+  const outFile = args[1] || `whitenoise-${new Date().getTime()}.mp3`
+
+  // check appropriate CLI usage
+  if (!size || !outFile) {
+    throw new Error(
+      `Incorrect script usage for input size (${size}) and outFile (${outFile}).\nPlease follow the structure 'node generateWhiteNoise.js <size> <outFile>'`
+    )
+  }
+
+  return { size, outFile }
+}
+
+/**
+ * Generates mp3 file using ffmpeg from given filesize in KiB
+ *
+ * The generated mp3 file is guaranteed to be the same duration for same size.
+ * It is also guaranteed to be random and unique across multiple runs.
+ *
+ * The output file is stored at the directory from which the script was called.
+ */
+function generateWhiteNoise(size, outFile) {
+  return new Promise((resolve, reject) => {
+    const process = spawn(ffmpeg, [
+      '-f', // flag for below
```

Comment:
I think we can go a little more in depth about what these specific flags do. here's an example of how we commented out ffmpeg.js in our code base https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/ffmpeg.js#L18-L29. could you add comments specific like this?

Reply:
ideally, someone like I should be able to read this script and know enough about how this all works without having the refer to the links. If I want to know more than the surface level comments, I can go more in depth through the link

Reply:
The problem is that the lines are broken up here, `-f lavfi` and `-i anoisesrc=d=size` are supposed to be single line in the command line, but are broken here.

I have updated accordingly. Let me know, thanks!